packages/server/modules/blobstorage/services.js

```diff
@@ -154,6 +154,16 @@ const updateBlobMetadata = async (streamId, blobId, updateCallback) => {
   return { blobId, fileName, ...updateData }
 }
 
+const getFileSizeLimit = () => {
+  //100Mb default
+  let sizeLimit = 100
+  const suppliedSize = parseInt(process.env.FILE_SIZE_LIMIT_MB)
```

Comment:
I try to keep all env variables in `./modules/core/helpers/envHelper.js` so they're never accessed directly from process.env

Reply:
fair enough, i do not like cross module dependency tangles, so moving the env helper to shared.